manta-js/package/src/sdk.interfaces.ts

```diff
@@ -48,16 +53,16 @@ export interface IMantaPrivateWallet {
   loggingEnabled: boolean;
   parameters: any;
 
-  loadUserMnemonic():any;
+  loadUserSeedPhrase():any;
   loadAuthorizationContext():any;
   dropAuthorizationContext():any;
-  dropUserMnemonic():any;
+  dropUserSeedPhrase():any;
   convertPrivateAddressToJson(address: string): any
   getNetworks(): any;
   getZkAddress(): Promise<Address>;
   getAssetMetadata(assetId: BN): Promise<any>;
   initalWalletSync(): Promise<boolean>;
-  walletSync(): Promise<boolean>;
+  walletSync(forceSync?: boolean): Promise<boolean>;
```

Comment:
Can you add some description in the PR about the interface changes.

Reply:
yeah, will update